usermanagment.go

```diff
@@ -73,13 +73,15 @@ type User struct {
 
 // UserProperties object
 type UserProperties struct {
-	Firstname     string `json:"firstname,omitempty"`
-	Lastname      string `json:"lastname,omitempty"`
-	Email         string `json:"email,omitempty"`
-	Password      string `json:"password,omitempty"`
-	Administrator bool   `json:"administrator,omitempty"`
-	ForceSecAuth  bool   `json:"forceSecAuth,omitempty"`
-	SecAuthActive bool   `json:"secAuthActive,omitempty"`
+	Firstname         string `json:"firstname,omitempty"`
+	Lastname          string `json:"lastname,omitempty"`
+	Email             string `json:"email,omitempty"`
+	Password          string `json:"password,omitempty"`
+	Administrator     bool   `json:"administrator,omitempty"`
+	ForceSecAuth      bool   `json:"forceSecAuth,omitempty"`
+	SecAuthActive     bool   `json:"secAuthActive,omitempty"`
+	Active            bool   `json:"active,omitempty"`
+	S3CanonicalUserID string `json:"s3CanonicalUserId,omitempty"`
```

Comment:
I assume this field was also forgotten in the swagger, but can be relied upon?

Reply:
No, it's in swagger, I didn't touch this field apart from indentation changes.

Reply:
OK. But it is a new field in the struct, so could you please mention it in the PR title or comment?

Reply:
You are right. I added this field. I can't remember. I was pretty sure I only added `Active`. But git proves me wrong. So yeah, I'll also adapt the unit test.